### DISCUSSION TITLE
Combine playlists - Problems

### DISCUSSION THREAD

**Discussion by Piratenkanon** [AzuraCast/AzuraCast#6260] [Q&A] [answered]
Hello community,
i have the following configuration in Playlists:
![playlist-01](https://user-images.githubusercontent.com/62739604/236620124-b23b2d35-532b-4eba-924e-2a4cfaa5c816.jpg)
AutoDJ (General rotation, weight 3) and Jingles (Once per 10 Songs) running perfect !!!!

But the remote URL doesn't start at the scheduled time !!
Configuration as follows:
![playlist-02](https://user-images.githubusercontent.com/62739604/236620525-bdff0752-1913-4cc2-a149-5b1bd0e50371.jpg)
![playlist-03](https://user-images.githubusercontent.com/62739604/236620529-9416ab90-743a-4a17-b320-064f559ddbaf.jpg)
![playlist-04](https://user-images.githubusercontent.com/62739604/236620536-5cae5d4d-da14-47e0-aeea-ecd77ae51bb0.jpg)

It also appears in the plan but does not launch.
![playlist-05](https://user-images.githubusercontent.com/62739604/236620572-5cb7ac8c-2199-4b94-be47-3efc275dfb1e.jpg)

WHAT IS WRONG HERE?

Thanks for the help in advance

**Comment by LordHelmchen666** [comment, marked as answer]
Try with turned off "Andere Songs unterbrechen, ...". Then it should work.

**Comment by Piratenkanon** [comment]
Thanks @LordHelmchen666  it work's!!!!!!!!! After restarting the Liquidsoap (AutoDJ)
